.github/workflows/demo_apps_build.yaml

```diff
@@ -0,0 +1,154 @@
+                    && west zephyr-export \
+                    && pip install -r scripts/requirements.txt \
+                    && source zephyr-env.sh \
+                    && west blobs fetch hal_telink \
+                    && ls -la \
+                    && pwd \
+                    && cd .. \
+                    && ls -la \
+                    && pwd
+
+            - name: Build Telink samples/basic/blinky app
+              run: |
+                    source .venv/bin/activate \
+                    && cd samples/basic/blinky \
+                    && west build -p -b tlsr9518adk80d
+
+            - name: Upload samples/basic/blinky artifacts
+              uses: actions/upload-artifact@v3
+              with:
+                    name: blinky_arfcts
+                    path: |
+                            samples/basic/blinky/build/zephyr/zephyr.bin
+                            samples/basic/blinky/build/zephyr/zephyr.elf
+
+            - name: Build Telink samples/bluetooth/peripheral_ht app
+              run: |
+                    source .venv/bin/activate \
+                    && cd samples/bluetooth/peripheral_ht \
+                    && west build -p -b tlsr9518adk80d
+
+            - name: Upload samples/bluetooth/peripheral_ht artifacts
+              uses: actions/upload-artifact@v3
+              with:
+                    name: peripheral_ht_arfcts
+                    path: |
+                            samples/bluetooth/peripheral_ht/build/zephyr/zephyr.bin
+                            samples/bluetooth/peripheral_ht/build/zephyr/zephyr.elf
+
+            - name: Build Telink samples/net/openthread/coprocessor app
+              run: |
+                    source .venv/bin/activate \
+                    && cd samples/net/openthread/coprocessor \
+                    && west build -p -b tlsr9518adk80d -- -DOVERLAY_CONFIG=overlay-rcp.conf \
+
+            - name: Upload samples/net/openthread/coprocessor artifacts
+              uses: actions/upload-artifact@v3
+              with:
+                    name: coprocessor_arfcts
+                    path: |
+                            samples/net/openthread/coprocessor/build/zephyr/zephyr.bin
+                            samples/net/openthread/coprocessor/build/zephyr/zephyr.elf
+
+            - name: Build Telink samples/net/sockets/echo_server app
+              run: |
+                    source .venv/bin/activate \
+                    && cd samples/net/sockets/echo_server \
+                    && west build -p -b tlsr9518adk80d -- -DOVERLAY_CONFIG=overlay-ot.conf \
+
+            - name: Upload samples/net/sockets/echo_server artifacts
+              uses: actions/upload-artifact@v3
+              with:
+                    name: echo_server_arfcts
+                    path: |
+                            samples/net/sockets/echo_server/build/zephyr/zephyr.bin
+                            amples/net/sockets/echo_server/build/zephyr/zephyr.elf
```

Comment:
Here is typo "amples" instead of "samples"

Reply:
Done